src/commands/receipt.ts

```diff
@@ -0,0 +1,49 @@
+import type { Arguments, CommandBuilder } from "yargs";
+import { Wallet, providers } from "ethers";
+import { connect, ConnectionOptions, resultsToObjects } from "@tableland/sdk";
+import yargs from "yargs";
+
+type Options = {
+  // Local
+  hash: string;
+
+  // Global
+  privateKey: string;
+  host: string;
+};
+
+export const command = "receipt <hash>";
+export const desc =
+  "Get the receipt of a chain transaction to know if it was executed, and the execution details";
+
+export const builder: CommandBuilder<Options, Options> = (yargs) =>
+  yargs.positional("hash", {
+    type: "string",
+    description: "Transaction hash",
+  }) as yargs.Argv<Options>;
+
+export const handler = async (argv: Arguments<Options>): Promise<void> => {
+  const { hash, host, privateKey } = argv;
+  const options: ConnectionOptions = {};
+  if (privateKey) {
+    // FIXME: This is a hack due to js-tableland's restrictive use of provider
+    // See: https://github.com/tablelandnetwork/js-tableland/issues/22
```

Comment:
here too

Reply:
Same same.

Reply:
Nope, not needed because I fixed in @tableland/sdk!